weblate/api/serializers.py

```diff
@@ -177,7 +177,7 @@ class Meta(object):
             'branch', 'filemask', 'template', 'new_base', 'file_format',
             'license', 'license_url', 'web_url', 'url',
             'repository_url', 'translations_url', 'statistics_url',
-            'lock_url', 'changes_list_url', 'new_lang',
+            'lock_url', 'changes_list_url', 'new_lang', 'push',
```

Comment:
This needs more care to be consistent with VCS view permissions:

* Add wiping to `to_representation` below
* Apply credentials cleanup in case it is enabled (it is done in the `RepoURLField`)

Reply:
- Done. I add wiping push link to `to_representation`
- Done. I add credentials cleanup and rewrite `test_create_component` to check the response does not contain secret data in 'push' field